recipes/soci/all/cmake/SOCI-override.cmake

```diff
@@ -0,0 +1,23 @@
+include(SOCI)

```

Comment:
Please use conan components to describe these.
See https://docs.conan.io/en/latest/creating_packages/package_information.html?highlight=component#using-components

Right now, conan will not use this information to create cmake targets, but that is on the roadmap.